.github/workflows/extension_deployment.yml

```diff
@@ -26,12 +28,3 @@ jobs:
       - name: Zip Build Files
         run: yarn build-zip
         working-directory: ${{env.WORKING_DIRECTORY}}
-      - name: Deploy to Extension Store
```

Comment:
this doesn't seem like a good idea

Reply:
no, I was just trying xD

Reply:
the idea was just trying the part it fails, obviously. And then add that piece back again after fixing. We are probably going to need this actually, the new `node/setup`